### PR TITLE
New package: aardappel.lobster version 2024.0

### LINKED ISSUE #156085: [Package Request]: Lobster
### How can we help?

I would like someone else to build the manifest.

### Please read and ensure the following

- [X] The installer meets the above requirements

### Please provide the following information

Download Page Url: https://github.com/aardappel/lobster
Publisher: https://github.com/aardappel
Package Name: Lobster
Description: The Lobster Programming Language
Package Version: v2024.0
Installer URL: https://github.com/aardappel/lobster/releases/download/v2024.0/lobster_windows_release.zip
Official webpage: https://www.strlen.com/lobster/
Official repository: https://github.com/aardappel/lobster

### PR DESCRIPTION
**Pull request by Exorcism0666**
Checklist for Pull Requests
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?
- Resolve #156085

Manifests
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/156215)

### CODE REVIEW

**Comment by Exorcism0666** [comment]
## For moderators

This account is automated by Github Actions and the source code was created by CoolPlayLin. If you have any questions about any pull request, don't hesitate to ping Exorcism0666.
>[!warning]
>With the new version of Komac, numerous Pull Requests may contain bugs or unwanted elements. If such issues arise, please tag russellbanks so that these bugs can be addressed in future updates.

>[!important]
>Please carefully review these Pull Request before merging. If it is a Pull Request for removing incorrect content and the URLs are issue, free upon manual checking, please close this Pull Request directly. (It is best to inform Exorcism0666 of the closure.)

良い一日をお過ごしください！

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> The file was not found. Verify git path must match with the casing of the file in disk. Not found file on 'manifests\a\aardappel\lobster\2024.0\aardappel.lobster.installer.yaml'

**Comment by Trenly** [comment]
The PackageIdentifier will need to be case-sensitive with the existing Publisher - `Aardappel`

File: manifests/a/aardappel/lobster/2024.0/aardappel.lobster.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: aardappel.lobster
```
Comment (stephengillie) [inline manifests/a/aardappel/lobster/2024.0/aardappel.lobster.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: Aardappel.lobster
```
Please update file name and path to match this change.

File: manifests/a/aardappel/lobster/2024.0/aardappel.lobster.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: aardappel.lobster
```
Comment (stephengillie) [inline manifests/a/aardappel/lobster/2024.0/aardappel.lobster.installer.yaml:4]:
```suggestion
PackageIdentifier: Aardappel.lobster
```
Please update file name and path to match this change.

File: manifests/a/aardappel/lobster/2024.0/aardappel.lobster.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.6.0.schema.json
+
+PackageIdentifier: aardappel.lobster
```
Comment (stephengillie) [inline manifests/a/aardappel/lobster/2024.0/aardappel.lobster.yaml:4]:
```suggestion
PackageIdentifier: Aardappel.lobster
```
Please update file name and path to match this change.

**Review by stephengillie — CHANGES_REQUESTED** [review]
Hi @Exorcism0666,

> The PackageIdentifier will need to be case-sensitive with the existing Publisher - `Aardappel`

```PowerShell
PS C:\ManVal> Find-WinGetPackage Aardappel

Name Id             Version    Source
---- --             -------    ------
Cube Aardappel.Cube 2005.08.29 winget
```

**Pull request closed without merging**